### PR TITLE
Add work area sensors for Husqvarna Automower

### PR DESCRIPTION
**Pull request by Thomas55555**
## Proposed change

Add work area sensors for mowers with [systematic mowing](https://www.husqvarna.com/uk/support/husqvarna-self-service/what-is-systematic-mowing-ka-70248/)

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/126931
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

## Summary by CodeRabbit

- **New Features**
	- Introduced integration with Husqvarna Automower for Home Assistant.
	- Added various entities including binary sensors, device tracker, and control actions for enhanced mower management.
	- Enabled scheduling and performance monitoring through new calendar and sensor entities.
	- Provided actions for overriding schedules and managing work areas.
	- Detailed documentation on OAuth2 credential configuration and setup process for the integration.

### CODE REVIEW

File: source/_integrations/husqvarna_automower.markdown [hunk 1]
```diff
@@ -161,6 +161,11 @@ The integration will create the following sensors:
 - Total searching time
 - Work area (if available). For example: *My lawn*, *Front lawn*, *Back lawn*
 
+For each work area with activated systematic mowing these sensors are created:
+
+- Progress
```
Comment (c0ffeeca7) [inline source/_integrations/husqvarna_automower.markdown]: Progress in percent?
Last time completed: is that something like "2 days ago"?

Comment (Thomas55555) [reply source/_integrations/husqvarna_automower.markdown]: > Progress in percent? 

Yes

> Last time completed: is that something like "2 days ago"?

Yes, it's a datetime object. So the frontend renders something like that.

Comment (c0ffeeca7) [inline source/_integrations/husqvarna_automower.markdown]:
```suggestion
- Progress (in percent)
```

Comment (c0ffeeca7) [reply source/_integrations/husqvarna_automower.markdown]: :pray: thank you

**Review by c0ffeeca7 — APPROVED** [review]
Thank you, @Thomas55555 :+1:

**Pull request merged**